docs/config/configuring-nodes-storage.rst

```diff
@@ -49,5 +49,10 @@ The following pages describe how to configure the backends currently supported b
    openstack-configuration
    aws-configuration
    emc-configuration
+   vmware-configuration
```

Comment:
In particular as ZFS says "(EXPERIMENTAL)" that indicates we support the third party drivers more than ZFS. In reality we don't test against these third party drivers. cc @ferrantim .